libs/langchain/langchain/agents/agent_toolkits/__init__.py

```diff
@@ -59,16 +65,18 @@
     "ZapierToolkit",
     "create_csv_agent",
     "create_json_agent",
+    "create_multion_agent",
     "create_openapi_agent",
     "create_pandas_dataframe_agent",
     "create_pbi_agent",
     "create_pbi_chat_agent",
     "create_python_agent",
-    "create_multion_agent",
+    "create_retriever_tool",
     "create_spark_dataframe_agent",
     "create_spark_sql_agent",
     "create_sql_agent",
     "create_vectorstore_agent",
     "create_vectorstore_router_agent",
     "create_xorbits_agent",
+    "create_conversational_retrieval_agent",
```

Comment:
outside scope of this PR

The namespacing is a bit surprising to me -- agent_toolkits sounds like it should only expose toolkits (collections of tool groups), but it's also exposing pre-built agents